yb-voyager/src/namereg/namereg.go

```diff
@@ -290,6 +288,19 @@ schema1.foobar, schema1."foobar", schema1.FooBar, schema1."FooBar", schema1.FOOB
 (fuzzy-case-match) schema1.fooBar, schema1."fooBar"
 */
 func (reg *NameRegistry) LookupTableName(tableNameArg string) (sqlname.NameTuple, error) {
+
+	sourceObjectNameMap := reg.SourceDBTableNames
+	targetObjectNameMap := reg.YBTableNames
+	if len(lo.Keys(reg.SourceDBSequenceNames)) > 0 {
```

Comment:
Is there any use-case for having to lookup both table names and sequence names together? should we have a separate function instead - `LookupSequenceName` ? Makes it more clear.

Reply:
Hmm, there is no such use case for having the same function, just the difference is which map we are using to look up, maybe I can change it to a separate function LookupSequenceName let me see if its a lot of change

Reply:
Not making this change as I realised if I separate out the function it will be a breaking change because we are having the separate map for both and with this same function I handled by combining both the map so that should it will work even in upgrade.